insights/parsers/findmnt.py

```diff
@@ -0,0 +1,79 @@
+    """Parse output of ``findmnt -lo+PROPAGATION``.
+
+    Typical output of ``findmnt -lo+PROPAGATION`` command is::
+
+        TARGET                                                          SOURCE                                FSTYPE          OPTIONS                                                                       PROPAGATION
+        /sys                                                            sysfs                                 sysfs           rw,nosuid,nodev,noexec,relatime,seclabel                                      shared
+        /proc                                                           proc                                  proc            rw,nosuid,nodev,noexec,relatime                                               shared
+        /dev                                                            devtmpfs                              devtmpfs        rw,nosuid,seclabel,size=8035516k,nr_inodes=2008879,mode=755                   shared
+        /sys/kernel/security                                            securityfs                            securityfs      rw,nosuid,nodev,noexec,relatime                                               shared
+        /dev/shm                                                        tmpfs                                 tmpfs           rw,nosuid,nodev,seclabel                                                      shared
+        /run/netns                                                      tmpfs[/netns]                         tmpfs           rw,nosuid,nodev,seclabel,mode=755                                             shared
+        /netns                                                          tmpfs[/netns]                         tmpfs           rw,nosuid,nodev,seclabel,mode=755                                             shared
+        /run/netns/qdhcp-08f32dab-927e-4a61-933d-57d425827b57           proc                                  proc            rw,nosuid,nodev,noexec,relatime                                               shared
+        /run/netns/qdhcp-fd138c0a-5ec7-44f8-88df-0501c4c7a968           proc                                  proc            rw,nosuid,nodev,noexec,relatime                                               shared
+
+
+    Examples:
+
+        >>> output.search_target('shm') == [{'target': '/dev/shm', 'source': 'tmpfs', 'fstype': 'tmpfs', 'options': 'rw,nosuid,nodev,seclabel', 'propagation': 'shared'}]
+        True
+        >>> len(output.target_startswith('/run/netns')) == 3
+        True
+        >>> output.target_startswith('/run/netns')[0].get('propagation', None) == 'shared'
+        True
+
+    Attributes:
+        cols (list): List of key value pair derived from the command.
+
+        keywords (list): keywords present in the command, each
+                        keyword is converted to lowercase.
+    """
+    def parse_content(self, content):
+        self.cols = []
+        self.keywords = []
+        if not content:
+            return
```

Comment:
Added docstring.

Reply:
The docstring is not enough. 

```None: Returns ``None`` if the output is empty.```

 As `None` Parser is meanless for any `@condition` in rules, it's not recommended to generate a Parser object for empty collections, unless it (the empty output) is required by a specific rule. 
E.g.
```python
@condition(FindmntPropagation)
def check_empty_findmnt_propagation(fp):
      # unless the empty 'findmnt_lo_propagation' is meaningful to this `@condition`, it should be skipped.
```

To avoid anybody removing the on purpose `None` in the future, you should explain the reason why leave it as None instead of raising a `Skip` for empty.

Reply:
@psachin - any update

Reply:
anything unclear?

Reply:
No